components/filter/demo/filter.html

```diff
@@ -11,7 +11,7 @@
 		import '../../filter/filter-dimension-set.js';
 		import '../../filter/filter-dimension-set-value.js';
 	</script>
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1.0">
```

Comment:
I had to add that minimum scale because mobile emulators weren't working properly with the filter demo page (ie. there was a horizontal scroll bar being shown rather than using the correct scaling)